### PR TITLE
Feat/new scare math

### LINKED ISSUE #184: Change: Scare Math!
Criteria:
afbeelding

### PR DESCRIPTION
**Pull request by hassanmohamed2003**
## Description
added falloff, interact/throwing objects gets less scary the further the distance is with NPC and scaring npcs near realtor is half as effective

## Setting up testing environment
1. In the project window navigate to Project/Assets/Scenes and open the "Realtor Scene" scene.
2. Press Play.

#### Realtor investigation
Criteria:
- [x] When throwing an object from far NPC gets less scared
- [x] Added scare when object breaks
- [x] When NPC is in realtors detection range all fear gets halved
## Checklist
_These checks should always be true for your pull request_
- [x] My pull request is for one story/feature.
- [x] Every individual commit in this pull request is logical.
- [x] All code, documentation, commits and player seen texts are in English.
- [x] I have deleted unused / unnecessary code.
- [x] If my edits need a change in documentation, then I have updated the documentation.
- [x] All code is in correspondence with the code conventions.

### CODE REVIEW

File: Assets/Scripts/AI/NPC/FearHandler.cs [hunk 1]
```diff
@@ -73,11 +77,17 @@ public void Handle(ObservableObject observableObject, DetectedProperties detecte
                     _npcController.GetScared();
                 }
                 break;
+            case ObjectState.Broken:
+                _npcController.FearValue += _brokenAddition;
```
Comment (Jeroenvn) [inline Assets/Scripts/AI/NPC/FearHandler.cs]: this should be done in calculate fear value()

File: Assets/Scripts/AI/NPC/FearHandler.cs [hunk 2]
```diff
@@ -73,11 +77,17 @@ public void Handle(ObservableObject observableObject, DetectedProperties detecte
...
+                break;
             default:
                 return;
             }
        }
-
+        if (_npcController.SeenByRealtor)
+        {
+            fearToAdd /= 2;
+        }
```
Comment (Jeroenvn) [inline Assets/Scripts/AI/NPC/FearHandler.cs]: this should also be done in calculate fear value

**Review by Jeroenvn — DISMISSED** [review]
I think it looks good. It's kind of hard to test. There are some small issues in the npc controller that need to be adressed

**Review by dcsmit — APPROVED** [review]
Looks good!

**Pull request merged**